pom.xml

```diff
@@ -143,6 +143,10 @@
                 <artifactId>liberty-maven-plugin</artifactId>
                 <version>2.4.2</version>
                 <configuration>
+                    <install>
+                        <runtimeUrl>https://public.dhe.ibm.com/ibmdl/export/pub/software/websphere/wasdev/downloads/wlp/18.0.0.2/wlp-webProfile8-18.0.0.2.zip</runtimeUrl>
+                    </install>
```

Comment:
By default, it pulls the latest build which is 18.0.02. This line is not required.